movie_search/media.py

```diff
@@ -32,44 +35,73 @@ def lookup_id_in_data_by_query(data_dict, query):
 
 
 class MediaService(ABC):
+    
+    def fetch_media_details_from_api(self, media_type, tmdb_id):
+        media_data = tmdb_api_obj.get_data_from_endpoint(f"/{media_type}/{tmdb_id}")
+        videos_data = tmdb_api_obj.get_data_from_endpoint(f"/{media_type}/{tmdb_id}/videos")
+        return media_data, videos_data
+
+    def store_genres(self, genres_data, GenreModel):
+        genres = []
+        for genre_data in genres_data:
+            genre, created = GenreModel.objects.get_or_create(
```

Comment:
could also do one bulk create

Reply:
@bbelderbos I feel like we discussed this before, and there was a reason why it was a better solution to not do a bulk create. I will try to see if I find prior notes on that.

Reply:
Yep we did and I don't think it affects the bottomline so happy to leave it like this for now.